R/paginate_listing.R

```diff
@@ -34,62 +40,104 @@
 #' # Use `verbose = TRUE` to display more descriptive warnings or errors
 #' # paginate_listing(lsting, cpp = 80, lpp = 40, verbose = TRUE)
 #' @export
-paginate_listing <- function(lsting, lpp = 15,
-                             cpp = NULL,
+paginate_listing <- function(lsting,
+                             page_type = "letter",
+                             font_family = "Courier",
+                             font_size = 12,
+                             lineheight = 1,
+                             landscape = FALSE,
+                             pg_width = NULL,
+                             pg_height = NULL,
+                             margins = c(top = .5, bottom = .5, left = .75, right = .75),
+                             lpp,
+                             cpp,
```

Comment:
I think minsiblings and nosplitin should probably be removed as they dont' make sense in a listing context (For now, maybe we could look at them affecting key columns in the future, but they dont now), so please remove these arugments and then just hard code them below in the (soon to be renamed, see above) "`pag_tt_indices`" function where (`formatters::`)`pag_indices_inner` is called